src/conn.c

```diff
@@ -2146,9 +2146,21 @@ _evStopPolling(natsConnection *nc)
 
     nc->sockCtx.useEventLoop = false;
     nc->el.writeAdded = false;
-    s = nc->opts->evCbs.read(nc->el.data, NATS_EVENT_ACTION_REMOVE);
+    // The "write" event is added and removed as we write, however, we always
```

Comment:
👍 Right, matching the changes in `natsLibevent_Read`, thx for the explanation comment!